extras/docker/qns/run_endpoint.sh

```diff
@@ -48,9 +46,21 @@ run_quiche_client_tests () {
     # TODO: https://github.com/marten-seemann/quic-interop-runner/issues/61
     # remove this sleep when the issue above is resolved.
     sleep 3
-    $QUICHE_DIR/$QUICHE_CLIENT $QUICHE_CLIENT_OPT \
-        $CLIENT_PARAMS $REQUESTS >& $LOG
 
+    if [ "$TESTCASE" == "multiconnect" ]; then
+
+        for req in $REQUESTS
+        do
+            $QUICHE_DIR/$QUICHE_CLIENT $QUICHE_CLIENT_OPT \
+                $CLIENT_PARAMS $req >& $LOG
```

Comment:
```suggestion
                $CLIENT_PARAMS $req >> $LOG 2>&1
```

To prevent from getting log of last request only.